### DISCUSSION TITLE
Verified links do not open in app

### DISCUSSION THREAD

**Discussion by Richard-L** [community/community#135495] [Mobile]
So sorry if this has been asked too many times - a search didn't seem to yield any results.

As of 5-7 months ago, the syndication of GitHub URLs to open directly in the GitHub app is no longer working for me. Instead a browser window of the URL will be opened. Sadly this hampers my workflow severely, and it would be so nice to receive help on this 🙏

I'm experiencing the issue on a Google Pixel 7 Pro.![Screenshot_20240812-105132.png](https://github.com/user-attachments/assets/f544fdad-9501-42f1-bffc-bf9cd49d6307)

![Screenshot_20240812-105151.png](https://github.com/user-attachments/assets/8058528a-1524-4ecc-9706-fa8f4ddbda29)

When searching I had found a similar issue reported to the signal app, where something with a blank in an SHA checksum seemed to have been the culprit: https://github.com/signalapp/Signal-Android/issues/11797#issuecomment-1185894573

Curiously, when you long press a URL, it'll allow you to open it in the GitHub app. But tapping it will redirect to the browser, which is what's relevant mainly.

![Screenshot_20240812-105330.png](https://github.com/user-attachments/assets/8213da8f-6c9a-4175-93a7-05aa54a5131b)

Thanks a lot for any assistance.

**Comment by taki-on** [comment]
@Richard-L thank you for reaching out to us. I will share this issue with Android team.

**Comment by abdullahalhakimi** [comment]
Here are some potential solutions:

Default Browser Settings: Certain browsers, like Brave, now have app link handling turned off by default for privacy reasons. If you're using a privacy-focused browser, check if there’s an option in settings that controls how links are opened. In Brave, you can enable opening external app links under Settings -> Brave Shields & Privacy​ ([Brave Community](https://community.brave.com/t/android-external-app-links-in-brave-browser-are-completely-broken-unusable-and-not-opening/503022)).

System Settings: On Android, you can manage how links are opened through system settings. Go to Settings -> Apps -> Default Apps -> Opening Links and check if GitHub is set as the preferred app for GitHub links.

GitHub App Settings: Ensure that you have the latest version of the GitHub app installed and that it has the necessary permissions to handle its URLs.

I hope that helps

**Reply by Richard-L** [reply]
Appreciate your reply but it didn't help.

I use Chrome on mobile and checked through all settings. Couldn't spot anything that would seem relevant.

The android specific settings I've already checked and screenshotted above as you see.

I've looked through the GitHub settings a few times over the past months, but there's nothing that would seem like it solves this issue.
